### PR TITLE
gfortran: add page

### PR DESCRIPTION
**Pull request by Carltoffel**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** GNU Fortran (GCC) 12.2.1 20230201

Since GCC and GFortran are very similar, I just copy&pasted the GCC page and adapted it.

### CODE REVIEW

**Review by einverne — APPROVED** [review]
LGTM. Thank you for the new page.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution and welcome to tldr.

File: pages/common/gfortran.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# gfortran
+
+> Preprocess and compile Fortran source files, then assemble and link them together.
+> More information: <https://gcc.gnu.org/wiki/GFortran>.
+
+- Compile multiple source files into executable:
```
Comment (kbdharun) [inline pages/common/gfortran.md:6]:
```suggestion
- Compile multiple source files into an executable:
```

Comment (Carltoffel) [reply pages/common/gfortran.md]: Good point, then you may want to change it here, too:
https://github.com/tldr-pages/tldr/blob/main/pages/common/gcc.md

Comment (kbdharun) [reply pages/common/gfortran.md]: > Good point, then you may want to change it here, too: https://github.com/tldr-pages/tldr/blob/main/pages/common/gcc.md

Thanks, will make the change over there too.

**Pull request merged**